README.md

```diff
@@ -39,24 +39,28 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 <a name="prerequisites"></a>
 ### Prerequisites
 
-Minishift requires a hypervisor to run the virtual machine containing OpenShift. Depending on your
-host OS, you have the choice of the following hypervisors:
+Minishift requires a hypervisor to run the virtual machine containing OpenShift.
+Make sure that the hypervisor of your choice is installed and enabled on your system before you
+install Minishift.
+
+Depending on your host OS, you have the choice of the following hypervisors:
 
 * OS X
     * [xhyve](https://github.com/mist64/xhyve) (default), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion)
 * GNU/Linux
     * [KVM](https://minishift.io/docs/docker-machine-drivers.html#kvm-driver) (default) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 * Windows
-    * [Hyper-V](https://technet.microsoft.com/en-us/library/mt169373.aspx) (default) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-
-Make sure that the hypervisor is installed and enabled on your system before you install Minishift.
+    * [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) (default) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
 **Important:**
 
 - KVM and xhyve require specific installation and configuration steps.
 For more information, see [docker machine drivers installation](https://minishift.io/docs/docker-machine-drivers.html).
-- It is recommended to use `Virtualbox 5.1.12` or later on Windows to avoid the issue
-[Error: getting state for host: machine does not exist](https://minishift.io/docs/troubleshooting.html#error-getting-state-for-host-machine-does-not-exist)
+- To enable Hyper-V ensure that, after you [install Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v), you also [add a Virtual
+Switch](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_virtual_switch) using the Hyper-V Manager. Make sure that you pair the virtual switch with
+a physical network card which has Internet access.
```

Comment:
`physical network card which has Internet access` should be emphasized more. Probably make it bold.